ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -201,9 +201,12 @@ private[testtool] final class ParticipantTestContext private[participant] (
       .map(_.partyDetails)
 
   def listKnownParties(): Future[Set[Party]] =
+    listKnownPartyDetails().map(_.map(partyDetails => Party(partyDetails.party)).toSet)
+
+  def listKnownPartyDetails(): Future[Seq[PartyDetails]] =
```

Comment:
If after the previous suggestion this is unused, please remove it.

Reply:
Done